inc/util/cli_utils.hpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef UTIL_CLI_UTILS_HPP
+#define UTIL_CLI_UTILS_HPP
+
+#include <string>
+#include <vector>
+
+#include <boost/program_options.hpp>
+#include <boost/filesystem/operations.hpp>
+
+#include "util/logger.hpp"
+#include "vcf/validator.hpp"
```

Comment:
is this include needed? Ideally the `util` namespace won't depend on the `vcf` namespace